src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GermlineCNVNamingConstants.java

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+/**
+ * This class stores naming standards in the Germline CNV caller
```

Comment:
`{@link GermlineCNVCaller}`.  You might also add that these are coming from the python `gcnvkernel` package.

Reply:
Done